pkgs/applications/video/smplayer/default.nix

```diff
@@ -13,13 +14,12 @@ mkDerivation rec {
 
   dontUseQmakeConfigure = true;
 
-  preConfigure = ''
-    makeFlags="PREFIX=$out"
-  '';
+  makeFlags = [ "PREFIX=${placeholder "out"}" ];
 
   meta = {
     description = "A complete front-end for MPlayer";
-    homepage = http://smplayer.sourceforge.net/;
+    longDescription = "Either mplayer or mpv should also be installed for smplayer to play medias";
+    homepage = "https://www.smplayer.info";
```

Comment:
Has a consensus been reached on whether url should be quoted or not?

Reply:
The RFC is ongoing -> https://github.com/NixOS/rfcs/pull/45

Reply:
No change until an RFC has been accepted that says otherwise.

Reply:
Ok.
I actually wanted to update the URL itself, the quotes part was not intentional. I'll correct it.